### ISSUE TITLE
JMenu item spacing

### ISSUE THREAD

**Issue by srcarter3** [vincenzopalazzo/material-ui-swing#35]
It would be nice if all jmenu items were spaced evenly.
Currently, JCheckBoxMenuItems have more space than other items. 
![menu](https://user-images.githubusercontent.com/21114460/56386853-b0961700-61d7-11e9-92b2-88b53c8bc897.png)

**Comment by vincenzopalazzo** [comment]
Can you try if this bug exists in the new jar demo?

Thanks

**Comment by vincenzopalazzo** [comment]
Hi @srcarter3,

This bus was fixed whit new jar?

Good day

**Comment by zanderson9** [comment]
Doesn't look like it was fixed. The spacing between JMenuCheckBox items is larger than between regular JMenu items. Please see attached image: the spacing between items "Next Place", "Recent" is smaller than between "Restore Zoom...", "Restore Projection..."

![Spacing](https://user-images.githubusercontent.com/49419742/56446240-5750e500-62b6-11e9-9271-e85eefa63a3c.png)

**Comment by vincenzopalazzo** [comment]
I think the problem is with the icon because now they have the same borders with the normal items, what do you say if I try to increase the clearance between the normal items so as to put them equal?

**Comment by zanderson9** [comment]
I'm OK with that. The goal is to look "same" for all items, so let's try what you suggest.

**Comment by vincenzopalazzo** [comment]
I will work on it, thanks for the help

**Comment by vincenzopalazzo** [comment]
I closing this issue because was fixed but I waiting a feedback

**Comment by zanderson9** [comment]
@vincenzopalazzo, I am using the latest jar which includes the fix for JMenu items spacing. But I am not seeing the change, i.e. the spacing between regular items is noticeably smaller than between checkbox items. Attached is the screenshot of a drop-down menu taken when using the latest jar:
![menu](https://user-images.githubusercontent.com/49419742/56545633-0d256900-652d-11e9-8b8d-290e1e4ae8b5.png)

**Comment by vincenzopalazzo** [comment]
Hi @zanderson9, 
You have changed the UIManager and added different borders to it

```
RadioButtonMenuItem.border
CheckBoxMenuItem.border
MenuItem.border
```
If yes, these edges must be the same.

If you start the demo you can see that the borders and menu items are greater than those that appear on your screen.

Let's take a test, if you haven't changed the things in the UIDefault, try adding this border

`BorderFactory.createEmptyBorder(5, 0, 5, 0)`

if you change the effect I'll try to see if I find the problem of the difference between the demo and your project

**Comment by zanderson9** [comment]
Yes, now with the addition of the borders, I see the even spacing. Yeah!

**Comment by vincenzopalazzo** [comment]
Are was get the Older setting for border in your project or the bug in to my project?

**Comment by zanderson9** [comment]
I added Border settings, _BorderFactory.createEmptyBorder(5, 0, 5, 0)_,  **explicitly** into **our** project for each menu item: regular item, cgheckbox item and radiobutton item.

**Comment by vincenzopalazzo** [comment]
The same configuration as the project, I close the issue because it was solved in the 1.0 beta.

Thanks for your help @zanderson9
